src/migrations/apply.rs

```diff
@@ -819,6 +838,53 @@ async fn disable_ddl(cli: &mut Connection) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+async fn index_activate_concurrently(conn: &mut Connection) -> Result<(), anyhow::Error> {
+    let version = conn.get_version().await?;
+    let is_supported = version.specific()
+        >= ver::Specific {
+            major: 6,
+            minor: ver::MinorVersion::Minor(10),
+        };
```

Comment:
We aren't going to backport, so we shouldn't do a 6.10 version. Let's put in the actual nightly number once it is built.